client/asset/btc/btc.go

```diff
@@ -333,6 +333,9 @@ type BTCCloneCFG struct {
 	TxVersion func() int32
 	// ManualMedianTime causes the median time to be calculated manually.
 	ManualMedianTime bool
+	// LegacySentRawTx sents legacy raw tx which does not have positional fee
+	// rate param.
+	LegacySentRawTx bool
```

Comment:
OMG, I'm sorry I don't know why I said `LegacySentRawTx` when it's `LegacySentToAddr`